third_party/openvino/ngraph_c_api/src/ngraph_c_api.cpp

```diff
@@ -119,6 +119,48 @@ inline ngraph::op::PadType GetAutoPad(ngraph_auto_pad autoPad) {
   return auto_pad;
 }
 
+inline std::string GetGruActivation(ngraph_gru_activation activation) {
+  switch (activation) {
+    case Relu:
+      return "relu";
+    case Sigmoid:
+      return "sigmoid";
+    case Tanh:
+      return "tanh";
+    default:
+      assert(0);
+  }
+}
+
+inline std::vector<std::string> GetGruActivations(
```

Comment:
I think this function is unnecessary, how about removing?

Reply:
Done.